core/isolate.rs

```diff
@@ -348,19 +357,15 @@ impl Isolate {
     }
   }
 
-  fn respond(&mut self, maybe_buf: Option<&[u8]>) -> Result<(), JSError> {
+  fn respond(&mut self, maybe_buf: Option<&[u8]>) -> Result<(), ErrBox> {
```

Comment:
Please add a triple slash comment saying the ErrBox should always be downcast to CoreJsError.